### ISSUE TITLE
Allow dashboard users to export their data

### ISSUE THREAD

**Issue by Jackseed** [blockframes/blockframes#9010]
It's a value proposition and differentiation to enable users to export their data. This way they are not bind to use and can use the time spent to fill data in our apps for other usages. 
It can then be an incentive to fill their data in our apps. 

Since the feature exist for admins, I guess we have what we need to do it in not that much time and effort?

May need a design first though, with a new Export section. 

Feature open to discussion for both design & dev effort
cc @cams-goussale @2amarkina @phpgeekfr

**Comment by cams-goussale** [comment]
Good idea, yes. The initial [wireframes](https://www.figma.com/file/FhY2dJqeYrVTXh7Q7HA0m7/%E2%9C%85-%2B-%E2%9A%99%EF%B8%8F-Sellers'-Dashboard?node-id=53%3A8239) have `Export Analytics` buttons but they are on the `Title page report` and `Buyer page report` level. I think it would be better on the previous page, where all titles and all buyers are gathered:

/c/o/dashboard/home/title
/c/o/dashboard/home/buyer

**Comment by 2amarkina** [comment]
I agree that idea is good. On the design level, indeed, there is no special effort. 
To my mind, if on the teck side it doesn't take too much time, we could even place the export CTA both on the general pages (/c/o/dashboard/home/title and /c/o/dashboard/home/buyer) and on the specific pages as on the wireframes (specific title, specific buyer)

**Comment by cams-goussale** [comment]
[WIP]

After discussion with Vincent, it seems the scope of this issue is broader than just analytics. Here's some details.


## Catalog-dashboard

- [ ] as a seller, i should be able to export in excel my titles with their metadata
- [ ] i can export from `/myTitles` page or `/myAvails`
- [ ] the export contains the filtered results (if any filter set)
- [ ] title metadata to include in the excel export

`movie id`, `title`, `season number`, `internal ref`, `org`, `orgId`, `catalog status`, `catalog access`, `festival status`, `festival access`, `financiers status`, `financiers access`, `screeningCount`, `creation date`,  `last modification date`, `audience goals`, `audience targets`, `cast`, `content type`, `crew`, `directors`, `estimated budget`, `expected premiere`, `format`, `format quality`, `genres`, `custom genres`, `is original version available`, `key assets`, 

## Festival-dashboard

- [ ] as a seller, i should be able to export in excel my titles with their metadata
- [ ] i can export from `/myTitles` page
- [ ] the export contains the filtered results (if any filter set)
- [ ] title metadata to include in the excel export:

- [ ] as a seller, i should be able to export in excel my analytics
- [ ] i can export from `/c/o/dashboard/home/title` page and `/c/o/dashboard/home/buyer` page
- [ ] the export contains the filtered results (if any filter set)
- [ ] analytics to include in the excel export:

## Catalog-marketplace

- [ ] as a buyer, i should be able to export in excel the library
- [ ] i can export from `/c/o/marketplace/title` page
- [ ] the export contains the filtered results (if any filter set)
- [ ] title metadata to include in the excel export is the same as the pdf, ie. 

## Festival-marketplace

- [ ] as a buyer, i should be able to export in excel the library
- [ ] i can export from `/c/o/marketplace/title` page
- [ ] the export contains the filtered results (if any filter set)
- [ ] title metadata to include in the excel export is the same as the pdf, ie.

**Comment by cams-goussale** [comment]
To do: isolate export marketplace first. Then we'll do exports dashboard.
